Modules/Core/tests/Feature/SessionsControllerTest.php

```
@@ -105,6 +108,305 @@ public function it_displays_password_reset_form_with_token(): void
+        }
+
+        /** Assert */
+        // The 6th attempt should be rate limited
+        $this->expectException(ValidationException::class);
+        $this->expectExceptionMessage('auth.throttle');
+
+        $this->post(route('sessions.authenticate'), [
+            'email' => $email,
+            'password' => 'wrong-password',
+        ]);
+    }
```

PHPMD (2.15.0)

The method it_rate_limits_after_multiple_failed_attempts is not named in camelCase. (undefined)

(CamelCaseMethodName)